server/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
+    commonjs: true,
```

Comment:
Hi @Implausibility, I see that you chose commonjs, airbnb-base and no react plugins. Are these settings working for your linter and React files?

Reply:
Those are my settings for inside the server folder, where everything is just Node. My general project linter files include the React plugins and the 'import' syntax.

Reply:
Ok great. You did account for React.